src/Components/Search/SearchBar.tsx

```diff
@@ -428,3 +429,15 @@ export const SearchBarQueryRenderer: React.FC = () => {
     />
   )
 }
+
+export function getSearchTerm(location: Location): string {
+  const term = get(
+    qs,
+    querystring => querystring.parse(location.search.slice(1)).term,
+    ""
+  )
+  if (Array.isArray(term)) {
+    return term[0]
+  }
```

Comment:
This logic could also be moved right inside of the `get` function. It's useful for any old return value :)